cmd/hope/vm/image.go

```diff
@@ -36,8 +36,12 @@ var imageCmd = &cobra.Command{
 			return err
 		}
 
-		// Group hypervisors by engine
-		hypEngMap := map[string][]hypervisors.Hypervisor{}
+		type hypervisorBuildCopy struct {
```

Comment:
Kinda feels like this belongs in the hypervisor package. It's all based on hypervisor-specific properties.